test/dotnet.Tests/CommandTests/Project/Convert/DotnetProjectConvertTests.cs

```diff
@@ -26,7 +26,7 @@ public void SameAsTemplate()
         var csFile = Path.Combine(dotnetProjectConvert, "Program.cs");
         File.WriteAllText(csFile, """Console.WriteLine("Test");""");
 
-        new DotnetCommand(Log, "project", "convert", "Program.cs")
+        new DotnetCommand(Log, "project", "convert", "Program.cs", "--source=move")
```

Comment:
I had similar feedback in the implementation, but, `--source=move` doesn't feel very clear to me. I think perhaps a term like "original files" may work better than "source files", and options like `--move-original-files` and `--copy-original-files` (those two being mutually exclusive with each other, or, perhaps in future allowing patterns like `--move-original-files *.resx`)

Reply:
I think it's def hard to say how much customization to provide in the convert command itself, and how much to leave to user to clean up manually afterwards.

One thing that could potentially help is a "dry run" option a la `git clean`. Since, if users can review that, before changing the state of their working tree, they could run convert with several different options, potentially (depending on how many we put in), and once the dry run seems to suggest something they are comfortable with, they can finally commit to it.
```sh
> $ git clean -nx
Would remove msbuild.binlog
```

Reply:
I'm fine with making this simpler for now and just always leaving the source files intact so the user can choose to delete them or not themselves.